### DISCUSSION TITLE
Can't add my exterior temperature sensor to Versatile Thermostat

### DISCUSSION THREAD

**Discussion by commandeurbrin** [jmcollin78/versatile_thermostat#763] [Q&A] [answered]
I'm trying to use Versatile Thermostat to control my electric heaters with zigbee modules.

As I'm trying to create a new thermostat, I just can't add any external temp entity when asked.

I get this error message : "Entity [weather.maison](http://weather.maison/) belongs to domain weather, expected ['sensor', 'input_number']"

I can set any entity but this one and I don't get it.

I'm very new to Home Assistant so I'm sorry if it's a stupid question...

**Comment by jmcollin78** [comment, marked as answer]
Hello, the entity 'weather.maison' is not a sensor. You have to give a sensor entity or an input_number.

The Meteo France integration gives a temperature sensor directly.

**Comment by MBRetrofit** [comment]
Helpers - Template - Sensor -  value_template: "{{ state_attr('weather.yourweatherservice', 'temperature') }}"
